.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,40 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: Type:Bug
+assignees: ''
+
+---
+
+**Describe the bug**
```

Comment:
good idea by the bot, or did you intentilnally not use headings?

Reply:
ha, no, faut of my linter config. Let me change it.

Reply:
blame my linter. Updating.